SystemServices/SystemServices.cpp

```diff
@@ -4061,7 +4071,7 @@ namespace WPEFramework {
         uint32_t SystemServices::getLastFirmwareFailureReason(const JsonObject& parameters, JsonObject& response)
         {
             bool retStatus = true;
-            FwFailReason failReason = FwFailReasonNone;
+            string flReason ="";
```

Comment:
please remove changes in SystemServices

Reply:
This is correct change, verified with unit test case

Results from Arrisxg1v3 box, even failure reason is Invalid request, getting as None.
root@arrisxg1v3:~# curl -H "Authorization: Bearer `WPEFrameworkSecurityUtility | cut -d '"' -f 4`" --header "Content-Type: application/json" --request POST --data '{"jsonrpc":"2.0","id":"3","method": "org.rdk.System.2.getLastFirmwareFailureReason"}' http://127.0.0.1:9998/jsonrpc;
{"jsonrpc":"2.0","id":3,"result":{"failReason":"None","success":true}}root@arrisxg1v3:~#
root@arrisxg1v3:~#
root@arrisxg1v3:~# cat /opt/fwdnldstatus.txt
Method|xconf
Proto|
Status|Failure
Reboot|
FailureReason|Invalid Request
DnldVersn|
DnldFile|
DnldURL|https://dac15cdlserver.ae.ccp.xcal.tv:443/Images/AX013AN_5.6p4s1_VBN_sey-signed.bin
LastRun|2022-08-31 14:09:10
FwUpdateState|Failed
DelayDownload|0
root@arrisxg1v3:~#